cmds/generate_timestamp.go

```diff
@@ -0,0 +1,46 @@
+package cmds
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+var generateTimestampDoc = `
+Write the current UTC timestamp to the file 'testtrack/build_timestamp.txt' in a
+TestTrack project. This timestamp can be passed as a param by the TestTrack
+client when calling the split registry endpoint from the TestTrack server.
```

Comment:
Let's be a little more definitive on how we use this file. `This timestamp is used by TestTrack clients to request the version of the split registry active at the given time.`

Also, we can probably drop `in a TestTrack project` since that can be assumed.